### ISSUE TITLE
REST Query Collection Based on Custom Field

### ISSUE THREAD

**Issue by jomho** [cockpit-project/cockpit#12785]
I imagine this should be a simple question but I cannot find an answer. 

I am trying to query using the REST API for one **article** with my custom field **id** of **THIS_IS_IT**;

The issue is I receive an array of every article no matter how I configure the filter object. What am I doing wrong?

 ```
 const options = {
    url: `${config.cms.endpoint}collections/get/articles?token=${config.cms.apiKey}`,
    method: 'POST',
    encoding: 'utf-8',
    body: JSON.stringify({
      filter: { published: true, id: 'THIS_IS_IT' },
      sort: { _created: -1 },
    }),
    resolveWithFullResponse: false,
  };

  const articles = await request(options);
```

Thank you so much!

**Comment by marusak** [comment]
sorry, but I am afraid I do not follow. How is this related to Cockpit?

**Comment by jomho** [comment]
Sorry, let me rephrase...
Using the cockpit collections API, how do I query a set of collections by a property value? I have only been able to return every document in a collection. 

https://getcockpit.com/documentation/api/collections

**Comment by jomho** [comment]
I am extremely sorry, I am in the wrong cockpit repo.
